packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -353,6 +358,30 @@ const AccountTransferForm = ({
         resetConverter();
     }, [selected_to, selected_from, account_limits]); // eslint-disable-line react-hooks/exhaustive-deps
 
+    const is_mt5_restricted =
+        selected_from?.is_mt &&
+        selected_from?.status?.includes('poa_failed') &&
+        authentication_status?.document_status !== 'verified';
+    const poa_pending_msg = localize(
+        'You will be able to transfer funds between MT5 accounts and other accounts once your address is verified.'
+    );
+    const Mt5RestrictedMsg = () => (
+        <Localize
+            i18n_default_text='Please <0>resubmit</0> your proof of address to transfer funds between MT5 and Deriv accounts.'
+            components={[<Link key={0} to={routes.proof_of_address} className='link dark' />]}
+        />
+    );
```

Comment:
```suggestion

    const poa_pending_msg = localize(
        'You will be able to transfer funds between MT5 accounts and other accounts once your address is verified.'
    );
    
    const Mt5RestrictedMsg = () => (
        <Localize
            i18n_default_text='Please <0>resubmit</0> your proof of address to transfer funds between MT5 and Deriv accounts.'
            components={[<Link key={0} to={routes.proof_of_address} className='link dark' />]}
        />
    );
```
nit: best to space this out for readability